TrafficCapture/trafficCaptureProxyServerTest/build.gradle

```diff
@@ -40,38 +57,66 @@ dependencies {
     }
 }
 
+CommonConfigurations.applyCommonConfigurations(project)
 configurations {
     all {
-        resolutionStrategy.dependencySubstitution {
-            substitute module('org.apache.xmlgraphics:batik-codec') using module('org.apache.xmlgraphics:batik-all:1.15')
-        }
+        exclude group: 'org.apache.logging.log4j', module: 'log4j-slf4j-impl'
     }
 }
 
-docker {
-    name "nginx-simple"
-    files 'src/main/docker/nginx-simple/'
+
+def dockerFilesForExternalServices = [
+        "nginx": "nginx-perf-test-webserver"
+]
+
+// Create the static docker files that aren't hosting migrations java code from this repo
+dockerFilesForExternalServices.each { projectName, dockerImageName ->
+    task("buildDockerImage_${projectName}", type: DockerBuildImage) {
+        def hash = calculateDockerHash(projectName)
+        images.add("migrations/${dockerImageName}:$hash")
+        images.add("migrations/${dockerImageName}:latest")
+        inputDir = project.file("src/main/docker/${projectName}")
+    }
 }
-//
-//jmeter {
-//    tool {
-//        version = "5.5"
-//    }
-//}
-//
-//task runJMeter(type: de.qualersoft.jmeter.gradleplugin.task.JMeterRunTask) {
-//    jmxFile.set("Test.jmx")
-//    dependsOn(tasks.getByName("composeUp"))
-//}
-//
-//task jmeterReport(type: de.qualersoft.jmeter.gradleplugin.task.JMeterReportTask) {
-//    jmxFile.set("Test.jmx")
-//    dependsOn('runJMeter')
-//}
+
+def javaContainerServices = [
+        "trafficCaptureProxyServer": "capture_proxy",
+        "trafficCaptureProxyServerTest": "jmeter"
+]
+def baseImageProjectOverrides = [
+        "nginx": "nginx"
+]
+
+def createContainerTasks = { projectName, dockerImageName ->
+    def dockerBuildDir = "build/docker/${projectName}"
+    CommonUtils.copyArtifact(project, projectName)
+
+    CommonUtils.createDockerfile(project, projectName, baseImageProjectOverrides, dockerFilesForExternalServices)
+}
+
+javaContainerServices.each(createContainerTasks)
+
+(javaContainerServices).forEach { projectName, dockerImageName ->
+    def dockerBuildDir = "build/docker/${projectName}"
+    task "buildDockerImage_${projectName}"(type: DockerBuildImage) {
+        dependsOn "createDockerfile_${projectName}"
+        inputDir = project.file("${dockerBuildDir}")
+        images.add("migrations/${dockerImageName}:${version}")
+        images.add("migrations/${dockerImageName}:latest")
```

Comment:
Note for myself.  This is identical to code within dockerSolution, but it seems fair to replicate it since it is short and because a couple policies are embedded within it (like the dual image tags) that could vary by project.